src/Api/CategoriApi.jsx

```diff
@@ -0,0 +1,18 @@
+import{useEffect,useState} from "react"
+import axios from 'axios'
+
+const categoryApi = ()=>{
+    const [categories, setCategories] = useState([])
+  
+    useEffect(()=>{
+        axios("/api/categories")
+        .then((res)=> setCategories(res.data.categories))
+      },[])
```

Comment:
the code does the job but I would suggest to use **async await**  for the api calls.